internal/command/providers_schema.go

```diff
@@ -56,7 +56,7 @@ func (c *ProvidersSchemaCommand) Run(args []string) int {
 	var diags tfdiags.Diagnostics
 
 	// Load the backend
-	b, backendDiags := c.Backend(nil)
+	b, backendDiags := c.backend(".", arguments.ViewHuman)
```

Comment:
This command basically enforces JSON output.
```suggestion
	b, backendDiags := c.backend(".", arguments.ViewJSON)
```